src/main/java/sysc4806/graduateAdmissions/dto/TermDTO.java

```diff
@@ -28,6 +28,5 @@
     @Enumerated(EnumType.STRING)
     private Season season;
     @NotNull
-    @NotBlank
-    private String year;
+    private int year;
```

Comment:
I should mention that theres a ‘@min(value)’ annotation that can ensure this has a minimum value so that we dont have to explicitly do a check, but it’s optional.